.github/workflows/build-and-deploy.yml

```diff
@@ -53,7 +53,11 @@ jobs:
         env:
           AWS_ACCESS_KEY_ID: ${{ secrets.CI_AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.CI_AWS_SECRET_ACCESS_KEY }}
+          # AWS_SESSION_TOKEN is optional.
+          AWS_SESSION_TOKEN: ${{ secrets.CI_AWS_SESSION_TOKEN }}
+          CI_ASSUME_ROLE_ARN: ${{ secrets.CI_ASSUME_ROLE_ARN }}
```

Comment:
Yeah it's a bit unfortunate that GitHub Actions only has a notion of _secret_ env vars that can be dynamically overridden via their UI. This means to parameterize anything in the workflow file will require either adding the parameter as an env var or to hard-code it like. Hard-coding a value makes it hard to create a fork since the workflow file would have to be modified in the fork to allow it to run.